checkout/src/main/java/com/checkout/tokenization/mapper/response/CardTokenizationNetworkDataMapper.kt

```diff
@@ -38,7 +38,7 @@ internal class CardTokenizationNetworkDataMapper : TokenizationNetworkDataMapper
             productId = result.productId,
             productType = result.productType,
             billingAddress = result.billingAddress?.let { AddressEntityToAddressDataMapper().map(it) },
-            phone = result.phone?.let { PhoneEntityToPhoneDataMapper().map(it) },
+            phone = result.phone?.let { PhoneEntityToPhoneDataMapper(result.billingAddress?.country).map(it) },
```

Comment:
Something could be optomised here:
* The mapper here can be set as a local lazy variable, as each execution of `fromCardTokenizationResponse()` can use the same `PhoneEntityToPhoneDataMapper`, same for `AddressEntityToAddressDataMapper`
* Follow the `Mapper().map()` pattern, we should pass the variable in the `map` function. In this case, we should pass the `Phone` and `iso3166Alpha2`, which can be achieved by using a `Pair`. It could be something like this:

```kotlin
internal class PhoneEntityToPhoneDataMapper : Mapper<Pair<PhoneEntity, String?>, Phone> {

    override fun map(from: Pair<PhoneEntity, String?>): Phone = Phone(
        from.first.number,
        Country.getCountry(from.first.countryCode, from.second)
    )
}

// then in the `fromCardTokenizationResponse` we will have this
phone = result.phone?.let { PhoneEntityToPhoneDataMapper().map(it to result.billingAddress?.country) },
```

It doesn't change the functionality given the current implementation, but it fits better for the `mapper.map()` pattern.

Reply:
Thanks @jheng-hao-lin-cko will include in next PR.